eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -53,26 +53,29 @@ foreach ($res in $responses)
     continue
   }
 
+  LogDebug "Branch [ $branchName ] in repo [ $RepoId ] has no associated open Pull Request. "
   if ($LastCommitOlderThan) {
     if (!$res.object -or !$res.object.url) {
       LogWarning "No commit url returned from response. Skipping... "
       continue
     }
     try {
       $commitDate = Get-GithubReferenceCommitDate -commitUrl $res.object.url -AuthToken $AuthToken
-      if ($commitDate -and ($commitDate -gt $LastCommitOlderThan)) {
-        LogDebug "The branch $branch last commit date $commitDate is newer than the date $LastCommitOlderThan. Skipping."
+      if (!$commitDate -or ($commitDate -gt $LastCommitOlderThan)) {
+        LogDebug "No last commit date or the branch $branch last commit date [ $commitDate ] is newer than the date $LastCommitOlderThan. Skipping."
         continue
       }
+      
+      LogDebug "Branch [ $branchName ] in repo [ $RepoId ] has a last commit date [ $commitDate ] that is older than $LastCommitOlderThan. "
```

Comment:
Nit: I generally use single quotes for special values in strings vs. brackets with spaces, as it will make your log statement shorter and easier to read without horizontal scrolling in some situations.